includes/fixes-dates.php

```diff
@@ -15,7 +15,7 @@
 if ( get_locale() == 'fa_IR' && wpp_is_active( 'persian_date' ) ) {
 	add_filter( 'the_time', 'wpp_fix_post_time', 10, 2 );
 	add_filter( 'the_date', 'wpp_fix_post_date', 10, 2 );
-	add_filter( 'get_the_time', 'wpp_fix_post_date', 10, 2 );
+	add_filter( 'get_the_time', 'wpp_fix_post_time', 10, 2 );
```

Comment:
We use 'wpp_fix_post_date' instead 'wpp_fix_post_time' to prevent conflict in admin area.

Reply:
Which conflict do you mean?!
This is the main reason for zero time bug in posts list (`00:00:00 AM`) & duplicate dates (like attach image), According to my reviews.
In my short experiments, there was no conflicts, But it's better to test it yourself. (In the previous version, `wpp_fix_post_date` was used!)

![2021-12-05_194853](https://user-images.githubusercontent.com/89868505/144754781-23a035f5-e6b0-41e8-9fe4-0e90e6bed50e.png)

Reply:
Thanks for description, we test it again in new version.